src/spatial/explog.hpp

```diff
@@ -114,7 +105,16 @@ namespace pinocchio
       Vector3 tmp((R.diagonal().array() + cphi) * beta);
       res(0) = (R (2, 1) > R (1, 2) ? Scalar(1) : Scalar(-1)) * (tmp[0] > Scalar(0) ? sqrt(tmp[0]) : Scalar(0));
       res(1) = (R (0, 2) > R (2, 0) ? Scalar(1) : Scalar(-1)) * (tmp[1] > Scalar(0) ? sqrt(tmp[1]) : Scalar(0));
-      res(2) = (R (1, 0) > R (0, 1) ? Scalar(1) : Scalar(-1)) * (tmp[2] > Scalar(0) ? sqrt(tmp[2]) : Scalar(0));
+      res(2) = (R (1, 0) > R (0, 1) ? Scalar(1) : Scalar(-1)) * (tmp[2] > Scalar(0) ? sqrt(tmp[2]) : Scalar(0));    }
+    else
+    {
+      const Scalar t = ((theta > TaylorSeriesExpansion<Scalar>::template precision<3>())
+                        ? theta / sin(theta)
+                        : Scalar(1)) / Scalar(2);
+      res(0) = t * (R (2, 1) - R (1, 2));
+      res(1) = t * (R (0, 2) - R (2, 0));
+      res(2) = t * (R (1, 0) - R (0, 1));
+
```

Comment:
The changes in this file are cosmetic changes only, right ?

Reply:
They just change the order of the if statement. It then allows propagating NAN correctly in the computational flow, without setting 0 values as done previously throw the if/else statements.

Reply:
I see. Thanks for the explanation.